src/Tecan/CustomCommands/DispenseParameters.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Mll\LiquidHandlingRobotics\Tecan\CustomCommands;
+
+use Mll\LiquidHandlingRobotics\Tecan\BasicCommands\AspirateAndDispenseParameters;
+use Mll\LiquidHandlingRobotics\Tecan\Rack\Rack;
+
+final class DispenseParameters
+{
+    public Rack $rack;
+
+    /**
+     * @var array<int, int>
+     */
+    public array $dispensePositions;
+
+    /**
+     * @param array<int, int> $dispensePositions
+     */
+    public function __construct(Rack $rack, array $dispensePositions)
+    {
+        $this->rack = $rack;
+        $this->dispensePositions = $dispensePositions;
+    }
+
+    public function formatToAspirateAndDispenseParameters(): AspirateAndDispenseParameters
+    {
+        /**
+         * We use min and max of the dispense position as start and end.
+         * Exclusion of the not excluded wells will happen in the calling class.
+         */
```

Comment:
```suggestion
        // We use min and max of the dispense position as start and end.
        // Exclusion of the not excluded wells will happen in the calling class.
```

Same deal. I see this as a (well-written) comment for the following code, not a PHPDoc that is semantically attached to the following variable `$startPosition`.